pycombo/pyCombo.py

```diff
@@ -44,15 +45,24 @@ def execute(
     num_split_attempts : int, default 0
         Number of split attempts. If 0, autoadjust this number automatically.
     fixed_split_step : int, default 0
-        Step number to apply predifined split. If 0, use only random splits,
+        Step number to apply predefined split. If 0, use only random splits,
         if >0 sets up the usage of 6 fixed type splits on every fixed_split_step.
-    random_seedd : int, default 0
+    treat_as_modularity : bool, default False
+        Indicates if edge weights should be treated as modularity scores.
+        If True, the algorithm solves clique partitioning problem over the given graph,
+        treated as modularity graph (matrix).
+        `modularity_resolution` is ignored in this case.
+    return_modularity : bool, default True
+        Indicates if function should return achieved modularity score.
+    random_seed : int, default -1
         Random seed to use.
+        Negative number indicates using some internal default value that is based on time
```

Comment:
shall we perhaps use `None` for non-set random seed (and set it as default)?
I think this is the "numpy" style

Reply:
Good point, will look into that.

Reply:
Done.